From 5b0c86e806b1c5ad716c20721bbb64aa5fcf979f Mon Sep 17 00:00:00 2001
From: Andrei Warkentin <andrei.warkentin@intel.com>
Date: Mon, 26 Feb 2024 15:27:30 -0600
Subject: [PATCH 1/2] BaseTools/Scripts/SetupGit.py: make aware of FdtBusPkg

Signed-off-by: Andrei Warkentin <andrei.warkentin@intel.com>
---
 BaseTools/Scripts/SetupGit.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/BaseTools/Scripts/SetupGit.py b/BaseTools/Scripts/SetupGit.py
index 5f93124967bf..0e3a0f353f12 100644
--- a/BaseTools/Scripts/SetupGit.py
+++ b/BaseTools/Scripts/SetupGit.py
@@ -38,6 +38,9 @@ CONFDIR = os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__
                        'Conf')
 
 UPSTREAMS = [
+    {'name': 'FdtBusPkg',
+     'repo': 'https://github.com/intel/FdtBusPkg.git',
+     'list': 'andrei.warkentin@intel.com', 'prefix': 'FdtBusPkg'},
     {'name': 'edk2',
      'repo': 'https://github.com/tianocore/edk2.git',
      'list': 'devel@edk2.groups.io'},
-- 
2.34.1

